Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -196,6 +196,11 @@ extension BrowserViewController: WKNavigationDelegate {
 
                 webView.configuration.preferences.javaScriptEnabled = !domainForShields.isShieldExpected(.NoScript)
             }
+            // Reset the block alert bool on new host. 
+            if let newHost: String = url.host, let oldHost: String = webView.url?.host, newHost != oldHost {
```

Comment:
No need to unwrap all of those, since we're doing equality checks:

`if url.host != webview.url?.host {`